packages/ui-extensions-go-cli/core/core.go

```diff
@@ -145,6 +145,7 @@ type Extension struct {
 	Capabilities    Capabilities     `json:"capabilities" yaml:"capabilities,omitempty"`
 	Development     Development      `json:"development" yaml:"development,omitempty"`
 	ExtensionPoints []string         `json:"extensionPoints" yaml:"extension_points,omitempty" toml:"extension_points,omitempty"`
+	Categories 			[]string         `json:"categories" yaml:"categories,omitempty" toml:"categories,omitempty"`
```

Comment:
the go cli part is being rewritten in typescript (that should be live now) - so we will need to see where to add that in the node version

Reply:
oh I think for the time being we need to support both. We automatically enable the node version for shopify employees, but normal users still get the go version :)